### PR TITLE
Document how to report security issues

### PR DESCRIPTION
**Pull request by raboof**
And some general updates to CONTRIBUTING.md

### CODE REVIEW

**Review by patriknw — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -67,17 +67,17 @@ The steps below describe how to get a patch into a main development branch (e.g.
 The steps are exactly the same for everyone involved in the project (be it core team, or first time contributor).
 
 1. To avoid duplicated effort, it might be good to check the [issue tracker](https://github.com/akka/akka/issues) and [existing pull requests](https://github.com/akka/akka/pulls) for existing work.
-   - If there is no ticket yet, feel free to [create one](https://github.com/akka/akka/issues/new) to discuss the problem and the approach you want to take to solve it.
+   - If there is no ticket yet, you can optionally [create one](https://github.com/akka/akka/issues/new) to discuss the problem and the approach you want to take to solve it.
```
[the remark this answers is not in the document]
Comment (raboof) [reply CONTRIBUTING.md]: Yeah, tricky balance: I like that we don't require it, but indeed working on substantial features without first discussing it in an issue is "at your own risk"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -112,11 +112,9 @@ Using, for example: current.version 2.5.22, previous.version 2.5, milestone.vers
 1. Merge backport PR after validation (no need for full PR reviews)
 1. Close issue
 
-## sbt
+## Getting started with sbt
 
-Akka is using the [sbt](https://github.com/sbt/sbt) build system. So the first thing you have to do is to download and install sbt. You can read more about how to do that in the [sbt setup](http://www.scala-sbt.org/0.13/tutorial/index.html) documentation.
-
-Note that the Akka sbt project is large, so `sbt` needs to be run with lots of heap (1-2 GB). This can be specified using a command line argument `sbt -mem 2048` or in the environment variable `SBT_OPTS` but then as a regular JVM memory flag, for example `SBT_OPTS=-Xmx2G`, on some platforms you can also edit the global defaults for sbt in `/usr/local/etc/sbtopts`.
+Akka is using the [sbt](https://github.com/sbt/sbt) build system. So the first thing you have to do is to download and install sbt. You can read more about how to do that in the [sbt setup](https://www.scala-sbt.org/1.x/docs/Getting-Started.html) documentation.
```
[the remark this answers is not in the document]
Comment (raboof) [reply CONTRIBUTING.md]: exactly

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (patriknw) [reply CONTRIBUTING.md]: I agree with both of you. Good to have a discussion before a big PR arrives, but nice to not have the overhead of an issue for small changes, like doc improvements.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -560,6 +553,13 @@ existence of those docs.
 
 # Supporting infrastructure
 
+## Reporting security issues
+
+If you have found an issue in an Akka project that might have security
+implications, you can report it to <security@lightbend.com>. We will make
+sure those will get handled with priority. Thank you for your responsible
+disclosure!
+
```
[the remark this answers is not in the document]
Comment (patriknw) [reply CONTRIBUTING.md]: I just noticed that Github has something for this https://github.com/akka/akka/security/advisories, but we should stick with the Lighbend procedure.

[the remark this answers is not in the document]
Comment (patriknw) [reply CONTRIBUTING.md]: but we can also mention it in https://github.com/akka/akka/security/policy
could you follow up with that @raboof ?

**Pull request merged**